components/common/ui/AddMemberInput.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
+import { ChangeEvent, KeyboardEvent, useState } from 'react';
+import styled from 'styled-components';
+
+import EmailBox from './EmailBox';
+
+const AddMemberInput = () => {
+  const [emailValue, setEmailValue] = useState('');
+  const [emailList, setEmailList] = useState<string[]>([]);
+
+  // 입력된 값의 구분 key 여부
+  let isDivisionKey = false;
+
+  // input 입력시
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+    if (isDivisionKey) {
+      isDivisionKey = false;
+      console.log(isDivisionKey);
+    } else {
+      setEmailValue(value);
+    }
+  };
+
+  // 이메일 구분 key가 눌렸을 때
+  const handleInputKeyDown = (e: KeyboardEvent<HTMLInputElement>) => {
+    const { key } = e;
+    if (key === 'Enter' || key === ' ' || key === ',') {
+      setEmailList([...emailList, emailValue]);
```

Comment:
```suggestion
      setEmailList(prev => [...prev, ...emailList, emailValue]);
```

state 불변성을 지키기 위해 배열이나 객체는 spread를 사용합시다!